### PR TITLE
Code Quality: Filter out LogLevel.Information

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- Closes #

**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Opened Files ...
2. ...

### CODE REVIEW

File: src/Files.App/Utils/Logger/SentryLogger.cs [hunk 1]
```diff
@@ -23,15 +23,19 @@ public bool IsEnabled(LogLevel logLevel)
 		public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception? exception, Func<TState, Exception?, string> formatter)
 		{
 			// Unhandled exceptions are captured in AppLifecycleHelper.HandleAppUnhandledException
-			if (exception is null || exception.Data[Mechanism.HandledKey] is false)
+			if
+			(
+				exception is null ||
+				exception.Data[Mechanism.HandledKey] is false ||
+				logLevel == LogLevel.Information
```
Comment (hishitetsu) [inline src/Files.App/Utils/Logger/SentryLogger.cs:30]:
```suggestion
				logLevel <= LogLevel.Information
```
Trace and Debug should be filtered out as well.

File: src/Files.App/Utils/Logger/SentryLogger.cs [hunk 2]
```diff
@@ -23,15 +23,19 @@ public bool IsEnabled(LogLevel logLevel)
...
+			)
 				return;
 
 			var generalSettingsService = Ioc.Default.GetRequiredService<IGeneralSettingsService>();
 
 			var level = logLevel switch
 			{
 				LogLevel.Debug => SentryLevel.Debug,
-				LogLevel.Information => SentryLevel.Info,
```
Comment (hishitetsu) [inline src/Files.App/Utils/Logger/SentryLogger.cs:34]: I don't think this needs to be deleted in case you change the level of filtering.

Comment (yaira2) [reply src/Files.App/Utils/Logger/SentryLogger.cs]: It doesn't make much difference so I added it back.

File: src/Files.App/Utils/Logger/SentryLogger.cs [hunk 3]
```diff
@@ -23,7 +23,12 @@ public bool IsEnabled(LogLevel logLevel)
 		public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception? exception, Func<TState, Exception?, string> formatter)
 		{
 			// Unhandled exceptions are captured in AppLifecycleHelper.HandleAppUnhandledException
-			if (exception is null || exception.Data[Mechanism.HandledKey] is false)
+			if
+			(
+				exception is null ||
+				exception.Data[Mechanism.HandledKey] is false ||
+				logLevel <= LogLevel.Information
+			)
```
Comment (0x5bfa) [inline src/Files.App/Utils/Logger/SentryLogger.cs:31]:
```suggestion
			if(exception is null ||
				exception.Data[Mechanism.HandledKey] is false ||
				logLevel <= LogLevel.Information)
```

**Review by 0x5bfa — CHANGES_REQUESTED** [review]

File: src/Files.App/Utils/Logger/SentryLogger.cs (hunk 3 above)
Comment (yaira2) [reply src/Files.App/Utils/Logger/SentryLogger.cs:31]: Isn't it easier to read if all the conditions are aligned?

Comment (0x5bfa) [reply src/Files.App/Utils/Logger/SentryLogger.cs:31]: Worsen for me. I don't get the reason why brackets have its own line break.

**Review by hishitetsu — APPROVED** [review]
LGTM

**Pull request merged**